engine/runtime-with-polyglot/src/test/java/org/enso/interpreter/test/ForeignMethodInvokeTest.java

```diff
@@ -74,4 +75,38 @@ public void testInteropWithJavaScript() throws Exception {
     assertTrue("It is an array2", res2.hasArrayElements());
     assertEquals(12, res2.getArrayElement(2).asInt());
   }
+
+  @Ignore
+  @Test
+  public void testParallelInteropWithJavaScript() throws Exception {
+    var source = """
+        from Standard.Base import all
+
+        polyglot java import java.lang.Thread
+
+        foreign js js_array t f = \"\"\"
+            f(300)
```

Comment:
Calls `java.lang.Thread.sleep(300)` giving the rest of the test a chance to run in parallel. Alas, that's not supported by current `OtherContextGuestObject.sendImpl` and there is no way in GraalVM 23.1 to hook into that. As such the test is currently `@Ignore`d and I propose to track this as a future work on Enso as well as GraalVM side.